media-gfx/iscan/iscan-2.30.4.2-r5.ebuild

```diff
@@ -92,6 +93,9 @@ src_configure() {
 	append-cppflags -D_GNU_SOURCE	# needed for 'strndup'
 	replace-flags "-O[0-9s]" "-O1"	# fix selector box bug 388073
 
+	# bug #963199
+	append-cflags -std=gnu89
```

Comment:
I think this would work with gnu17, as it is only falling over `()` vs `(void)`.